twitter_ads/http.py

```diff
@@ -141,6 +141,7 @@ def __init__(self, code, headers, **kwargs):
             raw_response_body = self._raw_body
 
         try:
+            raw_response_body = str(raw_response_body)
```

Comment:
This probably doesn't need to be in the `try`, unless `str(..)` raises a `ValueError`, which I don't think it does? Not that big of a deal though.